empty/database/schema.sql

```diff
@@ -0,0 +1,51 @@
+
+DROP TABLE IF EXISTS "users";
+
+CREATE TABLE "users" (
+  id SERIAL PRIMARY KEY,
+  user_name VARCHAR(20) NULL DEFAULT NULL,
+  password VARCHAR NULL DEFAULT NULL,
+  flaps INTEGER NULL DEFAULT NULL,
+  deaths INTEGER NULL DEFAULT NULL
+);
+
+
+DROP TABLE IF EXISTS "high_scores";
+
+CREATE TABLE "high_scores" (
+  id SERIAL PRIMARY KEY,
+  score INTEGER NULL DEFAULT NULL,
+  user_id INTEGER NULL DEFAULT NULL
+);
+
+DROP TABLE IF EXISTS "death_coordinates";
+
+CREATE TABLE "death_coordinates" (
+  id SERIAL PRIMARY KEY,
+  x_pos INTEGER NULL DEFAULT NULL,
+  y_pos INTEGER NULL DEFAULT NULL,
+  user_id INTEGER NULL DEFAULT NULL
+);
+
+
+ALTER TABLE "high_scores" ADD FOREIGN KEY (user_id) REFERENCES "users" ("id");
```

Comment:
You may define foreign key references inline - I prefer this to the `ALTER TABLE` approach to keep it directly associated with the table.